### DISCUSSION TITLE
Intermittent "The specified resource does not exist" errors

### DISCUSSION THREAD

**Discussion by gsavit** [fastlane/fastlane#18965] [Q&A]
Hey all!

Wondering if anyone else has experienced this and/or has ideas for what might be going on here. Pretty sure something is up on the Apple side, but not certain.

I'm leveraging spaceship basically as middleware to make calls to the official ASC API. I've noticed that requests occasionally (and seemingly randomly) fail with this error message: _"The specified resource does not exist"_. Retrying the request usually is successful.

Some examples of errors we've seen:
- (Calling `app_version.update`) _"There is no resource of type 'appStoreVersions' with id <ID>"_ I checked after the fact, appStoreVersion model did exist with the given ID.
- (Calling `ConnectAPI.get_app_store_versions`) _"The specified resource does not exist - There is no resource of type 'AppsV1' with id '<ID>'"_ Same deal, checked and the models looked OK.

Thanks so much in advance for any thoughts, ideas, etc.!

Gabriel

**Comment by aphex3k** [comment]
I see this to intermittently. Right now I haven't had it work for the last 48 hours straight :(

**Comment by iBotPeaches** [comment]
I was trying to investigate that a few days ago and I when I gathered some stats - I had reports from users on 3 different forms of authentication (legacy u/p w/ 2fa, appstore api - team key, appstore api - user key)

Then I made calls when my own generated appstore api (team key) and never replicated anything.
